src/Raven.Server/Integrations/PostgreSQL/RqlQuery.cs

```diff
@@ -335,6 +349,7 @@ protected virtual void HandleSpecialColumnsIfNeeded(string columnName, Blittable
 
         public override void Dispose()
         {
+            GC.SuppressFinalize(this);
```

Comment:
How do we dispose the context if `IsNamedStatement` is `true` ?

Reply:
Answered in https://github.com/ravendb/ravendb/pull/15226#issuecomment-1308791819